cmd/celestia/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"os"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(fullCmd, lightCmd)
+}
+
+func main() {
+	err := run()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	// TODO(@Wondertan): In practice we won't need all INFO loggers from IPFS/libp2p side
```

Comment:
Yeah it was a bit too verbose - any way to just display the peering logs? (e.g. `Peer added / dropped`)